recipes/lsst-documenteer-pipelines/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "documenteer" %}  {# PyPI name #}
+{% set version = "0.5.6" %}
+
+package:
+  name: "lsst-documenteer-pipelines"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "06fbceb7324ed3b00f012b90b2da0e0aff9faab7957d6d7c5f09cc5a3069c9cf"
```

Comment:
Do you want this package to get a version bump if you bump documenteer on pypi? If yes, then leave this here. Otherwise, you should set the source to `path: .` or something like that.

Reply:
Yeah, my intention is to release all three packages in step and give them all the same version numbers.

Reply:
OK. Then a single package with more then one output is definitely the best way to go here. I can help you make it if you are not familiar. I am on the lsstc slack channel if you want to chat offline on this. I am guessing you might be too!

see the conda build docs here: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#outputs-section